extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -34,21 +33,19 @@ import com.mapbox.maps.logW
  * @see [The online documentation](https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#geojson)
  *
  */
-class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
-  private val workerHandler by lazy {
-    Handler(workerThread.looper)
+class GeoJsonSource : Source {
+  @Deprecated("Use builder instead", level = DeprecationLevel.WARNING)
```

Comment:
I'd suggest to use level Error to force users to use Builder. It'll still be source-incompatible change, breaking compilation but not runtime behaviour.